src/reducers/podReducer.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * ************************************
+ *
+ * @module  podReducer
+ * @description reducer for pod data
+ *
+ * ************************************
+ */
+
+import * as types from '../constants/actionTypes';
+
+const initialState = {
+   podNames: [],
+   podInfo: []
+};
+ 
+const podReducer = (
+  state = initialState, 
+  action
+  ) => {
+  const deepStateClone = JSON.parse(JSON.stringify(state));
+   
+  switch (action.type) {
+    case types.SET_POD_NAMES:
+      return {
+        ...state,
+        podNames: action.payload
+        }
+    case types.SET_POD_INFO:
+    return {
+      ...state,
```

Comment:
We discussed this not being necessary if we are replacing all of the items that are passed by reference

Reply:
Thanks for the clarity! I see - this won't cause side effects for the time being since in this `switch-case` we are replacing the only property in the state. 

However, in the future, if we add more properties to the state, this can cause side effects - better to update it now to reduce technical debt. Plus, all other cases use `deepStateClone`, this one should too for consistency.

Reply:
Let's talk about this in our meeting to clarify